cross_valid_total.py

```diff
@@ -0,0 +1,197 @@
+
+
+DATASET_DIR = '/users/data/d3m/datasets/training_datasets'
```

Comment:
It's not a good idea to push file paths to public repositories. We should make this a command line arg or an environment variable.